lib/fileHandler/index.js

```diff
@@ -12,11 +13,18 @@ const needle = require('needle');
 
 const AWS = require('aws-sdk');
 
-const REGION = 'us-west-2',
-	BUCKET = 'workflow-file-uploads',
+const {
+	AWS_LAMBDA_FUNCTION_MEMORY_SIZE,
+	CONNECTOR_MAX_ALLOCATED_RAM_MB,
+	CONNECTOR_FILE_REGION,
+	CONNECTOR_FILE_BUCKET
+} = process.env;
+
+const REGION = CONNECTOR_FILE_REGION || 'us-west-2',
+	BUCKET = CONNECTOR_FILE_BUCKET || 'workflow-file-uploads',
 	DEV_BUCKET = 'workflow-file-uploads-dev';
```

Comment:
Since you are adding options for the production bucket, I can't see the harm in adding the option for the dev bucket aswell.

Reply:
Added